scripts/microbe_stage/organelle.as

```diff
@@ -684,13 +687,23 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
         if (species !is null){
             this.speciesColour = species.colour;
             }
+
         // Our coordinates are already set when this is called
         // so just cache this
         this.cartesianPosition = Hex::axialToCartesian(q, r);
 
+        // Calcuate hexSize depending if species is baceteria or not
+        double hexSize = 0;
+        if(species.isBacteria) {
+            this.cartesianPosition /= 2;
+            hexSize = 0.375;
+        }
+        else
+            hexSize = HEX_SIZE;
+
         assert(organelleEntity == NULL_OBJECT, "PlacedOrganelle already had an entity");
 
-        Float3 offset = organelle.calculateCenterOffset();
+        Float3 offset = organelle.calculateCenterOffset(species);
```

Comment:
It would be much better to have `/ 2` here if the species is bacteria instead of moving the logic to the Organelle class, which shouldn't have to care about this detail

Reply:
Ok so i just check isBacteria one time and if yes i just divide everything needs to be divide one time, instead making different check inside different functions

Reply:
Yes

Reply:
Actually this `offset` variable is already only used for graphics. So this should *not* be scaled. Because the microbe render root node is scaled *everything* attached to it is automatically scaled.

Reply:
Thats also the reason why organelle @renderNode do not need to be scaled too?

Reply:
Yes, the organelle render nodes are scaled automatically because the microbe root node is scaled.